static/assets/fedidevs.css

```
@@ -20,7 +22,7 @@
     --table-header-background-color: #ffffff08;
     --table-stripe-background-color: #ffffff08;
 
-    --code-font: 16px/1.4 "SFMono-Regular", Menlo, Monaco;
+    --code-font: 14px/1.4 "Noto Sans Mono";
```

```suggestion
    --code-font: 16px/1.4 ui-monospace, Menlo, Monaco, "Cascadia Mono", "Segoe UI Mono", "Roboto Mono", "Oxygen Mono", "Ubuntu Monospace", "Source Code Pro", "Fira Mono", "Droid Sans Mono", "Courier New", monospace;
```

I've experienced the same issue (#2) on Linux and a quick web search pointed me toward this wonderful resource: https://qwtel.com/posts/software/the-monospaced-system-ui-css-font-stack/#the-monospaced-system-ui-css-font-stack

Seems to me like a better option than relying on external fonts?

---

lgtm!